cookbook/rule/general_information_on_rule_format.rst

```diff
@@ -0,0 +1,562 @@
+
+Available Actions List
+++++++++++++++++++++++
+
+Copy
+____
+
+This action enables to copy an attribute value into another.
+
+.. warning::
+
+    Source and target attributes have to be of the same type. If source attribute is empty, the value "empty" will also
+    be copied.
+
+Two parameters are required and four other are optional:
+ - from_field: code of the attribute to be copied
+ - from_locale : locale code of the value to be copied (optional).
+ - from_scope : channel code of the value to be copied (optional).
+ - to_field: attribute code the value will be copied into
+ - to_locale: locale code the value will be copied into (optional)
+ - to_scope: channel code the value will be copied into (optional).
+
+.. tip::
+
+    For instance, to copy description from en_US print channel to the en_US description e-commerce channel, action will
+    be defined as follows:
+
+        .. code-block:: yaml
+
+            actions:
+                - type:        copy
+                  from_field:  description
+                  from_locale: en_US
+                  from_scope:  print
+                  to_field:    description
+                  to_locale:   en_US
+                  to_scope:    ecommerce
+
+Set
+___
+
+This action enables assigning values to an attribute.
+
+Two parameters are required, two other are optional.
+ - field : attribute code.
+ - locale : local code for which value is assigned (optional).
+ - scope : channel code for which value is assigned (optional).
+ - value : attribute value
+
+.. tip::
+
+    For instance, to set the value "My very new description for purple tshirt" to description attribute in en_US locale,
+    for ecommerce channel, the action will be as follows:
+
+    .. code-block:: yaml
+
+        actions:
+            ­ type:   set
+              field:  description
+              locale: en_US
+              scope:  ecommerce
+              value:  My very new description for purple tshirt
+
+Add
+___
+
+This action allows to add values to a multiselect, a category or a collection.
+
+Two parameters are required, two others are optional.
+ - field : attribute code.
+ - locale : local code for which value is assigned (optional).
+ - scope : channel code for which value is assigned (optional).
+ - value : attribute value
+
+.. tip::
+
+    For instance, adding category "t-shirts" action will be as follows:
+
+    .. code-block:: yaml
+
+        actions:
+            - type: add
+              field: category
+              value:
+                - t-shirts
+
+Fields
+++++++
+
+Created
+_______
++--------------+----------------------+
+| Operator     | - =                  |
+|              | - >                  |
+|              | - <                  |
+|              | - BETWEEN            |
+|              | - NOT BETWEEN        |
+|              | - EMPTY              |
++--------------+----------------------+
+| Value        | dates format :       |
+|              | mm/dd/yyyy. If       |
```

Comment:
currently, date format is still `yyyy-mm-dd`. If PO wants to change that with the localization, we need to create a card